packages/component/src/Activity/SendStatus.js

```diff
@@ -31,10 +31,15 @@ const connectSendStatus = (...selectors) =>
   );
 
 const SendStatus = ({ activity }) => {
-  const [{ sendStatus: sendStatusStyleSet }] = useStyleSet();
+  const [{ sendStatus: sendStatusStyleSet, timestamp: timestampStyleSet }] = useStyleSet();
   const focusSendBox = useFocusSendBox();
   const postActivity = usePostActivity();
 
+  sendStatusStyleSet &&
+    console.warn(
+      'Web Chat: styleSet.sendStatus is being deprecated. Please use styleSet.timestamp. This deprecation migration will be removed on or after December 17, 2021.'
```

Comment:
Hmm I'm not sure about the naming here. timestamp can be considered a send status, but send status is not a timestamp. Should we flip the variable names, or come up with a new container name?

Reply:
I prefer flip too. Send status is broader than timestamp. 😉

Reply:
You guys are the worst